### DISCUSSION TITLE
How Do I Export a Pointcloud

### DISCUSSION THREAD

**Discussion by johko** [facebookresearch/pytorch3d#1450] [Q&A] [answered]
Hey,

maybe a stupid question: How do I export a Pointcloud() object? I want to save it as obj or ply (or whatever is best) but can't seem to figure out the right way to do it with pytorch3d

**Comment by bottler** [comment, marked as answer]
If `pcl` is a `Pointclouds` object containing one point cloud, you can save it as a ply with::

    from pytorch3d.io import IO
    IO().save_pointcloud(pcl, "output_pointcloud.ply")

We don't currently support obj or other formats for point clouds. If you have a batch of point clouds you have to split it and save each one to a separate file.

**Comment by johko** [comment]
Thank you @bottler 

I also came across this page: https://pytorch3d.org/docs/io and wondered why it did not work.
Now I know, the example on the docs page has `IO().save_point_cloud()` instead of `IO().save_pointcloud()` 
Maybe you can fix this :)
